x-pack/plugins/observability_solution/entity_manager/server/lib/entities/install_entity_definition.test.ts

```diff
@@ -5,14 +5,14 @@
  * 2.0.
  */
 
+import semver from 'semver';
```

Comment:
the tests were using the builtin definition instead of a mock definition, I've updated that which adds a lot of noise but there's a new test case for outdated version in there